src/ResourceManager/Network/Commands.Network/help/Add-AzureRmRouteFilterRuleConfig.md

```diff
@@ -35,6 +35,8 @@ The Add-AzureRmRouteFilterRuleConfig cmdlet adds a route filter rule to an Azure
 
 
 
+
```

Comment:
@EvgenyAgafonchikov `platyPS` will sometimes add these blank lines (I haven't determined why yet), so you can go ahead and revert the changes that add unnecessary lines that are autogenerated.